Automattic/wp-calypso
# Pricing Plans upgrade block UX tweaks (II)

#### Proposed Changes

This Pull Request implements the UI/UX tweaks discussed at 116-gh-Automattic/lighthouse-forums#issuecomment-1326692511.

This is a continuation of https://github.com/Automattic/wp-calypso/pull/70330

#### Testing Instructions






* Run `yarn dev --sync` in `apps/happy-blocks` to sync code to your sandbox
* Create a new topic, in the editor press `/` and search for "Upgrade" block.
* Verify the issues discussed in 116-gh-Automattic/lighthouse-forums#issuecomment-1326692511  have been resolved.


#### Pre-merge Checklist





Related to #
